miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -0,0 +1,36 @@
+package com.rakuten.tech.mobile.miniapp.js
+
+import android.webkit.JavascriptInterface
+import com.google.gson.Gson
+import com.rakuten.tech.mobile.miniapp.display.WebViewListener
+
+/** Bridge interface for communicating with mini app. **/
+abstract class MiniAppMessageBridge {
+    private lateinit var webViewListener: WebViewListener
+
+    /** Get provided id of mini app for any purpose. **/
+    abstract fun getUniqueId(callbackId: String)
```

Comment:
I think that the callback ID should be internal to the SDK: the implementing client shouldn't need to care about it.

On iOS, this function signature is like this: `fun getUniqueId(): String`, so I think Android should match.